types/sanctuary/index.d.ts

```diff
@@ -294,30 +294,32 @@ declare namespace Sanctuary {
     //  Parse
     parseDate(s: string): Maybe<Date>;
     parseFloat(s: string): Maybe<number>;
-    parseInt(p: Integer): (q: string) => Maybe<Integer>;
+    parseInt(radix: 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | 24 | 25 | 26 | 27 | 28 | 29 | 30 | 31 | 32 | 33 | 34 | 35 | 36): (s: string) => Maybe<Integer>;
```

Comment:
Instead of disabling max line length, it would be better to split it to span several lines

Reply:
Feel free to change this. :)

Reply:
Please fix this

Reply:
I added line breaks and enabled `max-line-length`.